rest-api-spec/src/main/resources/rest-api-spec/test/bulk/10_basic.yml

```diff
@@ -65,12 +65,9 @@
   - skip:
       version: " - 5.4.99"
       reason: confusing exception messaged caused by empty object fixed in 5.5.0
-      features: ["headers"]
 
   - do:
       catch: /Malformed action\/metadata line \[3\], expected FIELD_NAME but found \[END_OBJECT\]/
-      headers:
-        Content-Type: application/json
```

Comment:
why did you remove these? I'm afraid they were there for a reason. This may work for clients without the header as they always use json content-type anyway, but providing the header disables the content-type randomization that the java yaml test runner usually does.

Reply:
OK - I can revert this and add a `features` skip clause instead.  The problem was that it had headers, but hadn't declared the need for the headers feature.

Reply:
Ah, I see that the `headers` feature has already been added, but it's impossible to ignore without also adding parsing to skip it... which i shall duly do (and revert this change)

Reply:
you mean that you would expect two skip clauses here? or the reason also highlighting why the headers skip?

Reply:
ok sorry we were kind of typing at the same time, ignore my last message ;)

Reply:
I think I've fixed this in https://github.com/elastic/elasticsearch/pull/26896, but I might have forgot to cherry-pick that commit into the `6.0` branch, which [I did yesterday](https://github.com/elastic/elasticsearch/commit/12a128beb7f51151b852be38bc52e3107a202a82).